### PR TITLE
Introduce mobile tests for all feature tests 

### PR DESCRIPTION
**Pull request by eduardoj**
Before, only tests under `features/beta/` were tested also with the mobile viewport. After this pull request, all feature tests will be tested with two viewports: desktop and mobile.

Only tests that were failing with mobile were moved out from beta.

Some remaining tests inside `features/beta/` were not moved outside, as they were not failing. They will be migrated in a following pull request, to not make this one so difficult to review.

-----------------

If this PR requires any particular action or consideration before deployment,
please check the reasons or add your own to the list:

* [ ] Requires a database migration that can cause downtime. Postpone deployment until maintenance window[1].
* [ ] Contains a data migration that can cause temporary inconsistency, so should be run at a specific point of time.
* [ ] Changes some configuration files (e.g. options.yml), so the changes have to be applied manually in the reference server.
* [ ] A new Feature Toggle[2] should be enabled in the reference server.
* [ ] Proper documentation or announcement has to be published upfront since the introduced changes can confuse the users.

[1] https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org#when-there-are-migrations
[2] https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#you-want-real-people-to-test-your-feature

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -265,23 +265,16 @@ jobs:
...
             if test "$CIRCLE_NODE_INDEX" = 0; then
               message="Running feature tests"
-              circleci tests glob 'spec/features/**/*_spec.rb' | grep -v 'spec/features/beta/' > $pickfile
             else
-              message="Running beta feature tests"
-              circleci tests glob 'spec/features/beta/**/*_spec.rb' > $pickfile
-            fi
-
-            if test "$CIRCLE_NODE_INDEX" = $((CIRCLE_NODE_TOTAL-1)); then
               message="Running beta feature tests for mobile"
```
Comment (dmarcoux) [inline .circleci/config.yml:272]:
```suggestion
              message="Running feature tests for mobile"
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -265,23 +265,16 @@ jobs:
           name: Run rspec feature tests
           command: |
             cd src/api
-            pickfile=log/pick.$CIRCLE_NODE_INDEX.list
 
             if test "$CIRCLE_NODE_INDEX" = 0; then
               message="Running feature tests"
```
Comment (dmarcoux) [inline .circleci/config.yml:270]:
```suggestion
              message="Running feature tests for desktop"
```

File: .circleci/config.yml (hunk 1 above)
Comment (eduardoj) [reply .circleci/config.yml]: Ops, applied.

File: .circleci/config.yml (hunk 2 above)
Comment (eduardoj) [reply .circleci/config.yml]: Applied, thanks.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**